Cargo.toml

```diff
@@ -52,7 +52,7 @@ serde = { version = "1.0.125", features = ["derive"] }
 syn = { version = "1.0", features = ["full"] }
 futures = "0.3"
 parking_lot = "0.12"
-tokio = { version = "1", features = ["io-util"] }
+tokio = { version = "1", features = ["full"] }
```

Comment:
This change is necessary to use `tokio::task_local`.

Reply:
It would be better not to depend on the implementation of `tokio::task_local` for the test. It looks like it was caused by having a nested macro call so the newly added macro to the test is enough.

Reply:
I commied https://github.com/rust-lang/rust-clippy/pull/9246/commits/b6c301055edc9d769bb7ac8e4b8bc083e31ac748